jguerrara_lru_cache.py

```diff
@@ -0,0 +1,26 @@
+class LRUCache:
+
+    def __init__(self, capacity):
+        self.vals = dict()
```

Comment:
It could be nice to pre size the dict since we know the max size, but this depends on the implementation details of the dict itself